crates/solvers/src/tests/legacy/market_order.rs

```diff
@@ -74,12 +74,12 @@ async fn quote() {
                     "decimals": 18,
                     "external_price": 1.0,
                     "internal_buffer": "1412206645170290748",
-                    "normalize_priority": null
+                    "normalize_priority": 1
                 },
                 "0xdef1ca1fb7fbcdc777520aa7f396b4e015f497ab": {
                     "accepted_for_internalization": true,
                     "alias": "COW",
-                    "decimals": 18,
+                    "decimals": null,
```

Comment:
Wasn't super clear to me why the `decimals` needed to change in the test.

Reply:
This was not strictly required but I thought since I added some [logic](https://github.com/cowprotocol/services/pull/1251#discussion_r1115001844) related to that I might as well add a little test for it.

Reply:
Makes sense!